### ISSUE TITLE
Support for MsgPack

### ISSUE THREAD

**Issue by tatey** [tatey/trace_privately#23]
This will mean a smaller payload. We should probably store the data in its raw binary form rather than a base64 encoded string too.

See https://github.com/CrunchyBagel/TracePrivately/pull/44

**Comment by HendX** [comment]
Agreed on storing it in binary. I just avoided this for now in the reference implementation because I'm dealing with PHP and my PHP skills are somewhat outdated - I just remember PHP having a lot of issues with dealing with things like binary data or UTF.
